SWI-cpp2.h

```diff
@@ -182,8 +210,10 @@ class PlFunctor : public WrappedC<functor_t>
 class PlAtom : public WrappedC<atom_t>
 {
 public:
-  PlAtom() : WrappedC<atom_t>() { } // Make constructor public
-  explicit PlAtom(atom_t v) : WrappedC<atom_t>(v) { }
+  PlAtom() // allow implicit "null" constructor, even though a bit dangerous
```

Comment:
Why do we want this?   Except for crashing it seems there is little useful you can do with this.

Reply:
See my other reply -- `rocksdb4pl.cpp` needs it to make `read_optdefs` easier to write. 

I've pushed a change to rocksdb4pl.cpp - please take a look at it (https://github.com/JanWielemaker/rocksdb/pull/15)